public/sass/components/_slate_editor.scss

```diff
@@ -16,7 +20,7 @@
   color: $text-color;
   background-color: $input-bg;
   background-image: none;
-  border: $panel-border;
+  border: $input-border;
```

Comment:
small not-relevant fix to have query field's border the same color as the rest of the inputs.